Remove Windows- and macOS-specific dependencies so that we don't have to
package them for Chromium OS.

Note that this patch applies to the normalized copy of Cargo.toml that lives
inside the crate, which is different to Cargo.toml committed to upstream git.

--- a/Cargo.toml.orig     2021-12-30 10:08:53.018349884 +1100
+++ b/Cargo.toml  2021-12-30 10:09:17.086436221 +1100
@@ -27,14 +27,6 @@
 [target."cfg(all(target_os = \"linux\", not(target_env = \"musl\")))".dependencies.libudev]
 version = "0.2.0"
 optional = true
-[target."cfg(any(target_os = \"ios\", target_os = \"macos\"))".dependencies.CoreFoundation-sys]
-version = "0.1.3"
-
-[target."cfg(any(target_os = \"ios\", target_os = \"macos\"))".dependencies.IOKit-sys]
-version = "0.1"
-
-[target."cfg(any(target_os = \"ios\", target_os = \"macos\"))".dependencies.mach]
-version = "0.2"
 [target."cfg(unix)".dependencies.bitflags]
 version = "1.0.4"

@@ -43,12 +35,6 @@

 [target."cfg(unix)".dependencies.nix]
 version = "0.16.1"
-[target."cfg(windows)".dependencies.regex]
-version = "1.0"
-
-[target."cfg(windows)".dependencies.winapi]
-version = "0.3.6"
-features = ["cguid", "commapi", "errhandlingapi", "fileapi", "guiddef", "handleapi", "minwinbase", "minwindef", "ntdef", "setupapi", "winbase", "winerror", "winnt"]
 [badges.appveyor]
 repository = "Susurrus/serialport-rs"
 service = "gitlab"
